themes/src/main/resources-community/theme/base/account/messages/messages_it.properties

```diff
@@ -71,83 +103,241 @@ date=Data
+identityProvider=Identity provider
+identityProviderMessage=Collegare il tuo account con gli identity provider che hai configurato
+socialLogin=Social Login
+userDefined=Definito dall''utente
+removeAccess=Rimuovi accesso
+removeAccessMessage=Devi concedere di nuovo l''accesso, se vuoi utilizzare l''account di questa applicazione.
+
+#Authenticator
+authenticatorStatusMessage=L''autenticazione a due fattori \u00e8 attualmente
+authenticatorFinishSetUpTitle=La tua autenticazione a due fattori
+authenticatorFinishSetUpMessage=Ogni volta che effettui l''accesso al tuo account Keycloak, ti verr\u00E0 richiesto di fornire il tuo codice di autenticazione a due fattori.
+authenticatorSubTitle=Imposta l''autenticazione a due fattori
+authenticatorSubMessage=Per incrementare la sicurezza del tuo account, attiva almeno uno dei metodi disponibili per l''autenticazione a due fattori.
+authenticatorMobileTitle=Autenticatore mobile
+authenticatorMobileMessage=Utilizza l''autenticatore mobile per ottenere i codici di verifica per l''autenticazione a due fattori.
+authenticatorMobileFinishSetUpMessage=L''autenticatore \u00e8 stato collegato al tuo telefono.
+authenticatorActionSetup=Set up
+authenticatorSMSTitle=Codice SMS
+authenticatorSMSMessage=Keycloak invier\u00E0 il codice di verifica al tuo telefono per l''autenticazione a due fattori.
+authenticatorSMSFinishSetUpMessage=I messaggi di testo vengono inviati a
+authenticatorDefaultStatus=Default
+authenticatorChangePhone=Cambia numero di telefono
+authenticatorBackupCodesTitle=Codici di backup
+authenticatorBackupCodesMessage=Ottieni i tuoi codici di backup a otto cifre
+authenticatorBackupCodesFinishSetUpMessage=Sono stati generati dodici codici di backup. Ognuno pu\u00f2 essere usato una sola volta.
+
+#Authenticator - Mobile Authenticator setup
+authenticatorMobileSetupTitle=Setup autenticatore mobile
+smscodeIntroMessage=Inserisci il tuo numero di telefono e ti verr\u00E0 inviato un codice di verifica.
+mobileSetupStep1=Installa un''applicazione di autenticazione sul tuo telefono. Sono supportate le applicazioni qui elencate.
+mobileSetupStep2=Apri l''applicazione e scansiona il codice QR:
+mobileSetupStep3=Inserisci il codice monouso fornito dall''applicazione e clicca Salva per completare il setup.
+scanBarCode=Vuoi scansionare il codice QR?
+enterBarCode=Inserisci il codice monouso
+doCopy=Copia
+doFinish=Termina
+
+#Authenticator - SMS Code setup
+authenticatorSMSCodeSetupTitle=Setup codice SMS
+chooseYourCountry=Scegli la tua nazione
+enterYourPhoneNumber=Inserisci il tuo numero di telefono
+sendVerficationCode=Invia il codice di verifica
+enterYourVerficationCode=Inserisci il codice di verifica
+
+#Authenticator - backup Code setup
+authenticatorBackupCodesSetupTitle=Setup backup codici
+backupcodesIntroMessage=Se non disponi pi\u00f9 del tuo telefono, puoi comunque accedere al tuo account attraverso i codici di backup. Conservai in un posto sicuro e accessibile.
```

Comment:
Se non disponi pi\u00f9 del tuo telefono, puoi comunque accedere al tuo account attraverso i codici di backup. Conservali in un posto sicuro e accessibile.